tensorflow/python/ops/array_ops.py

```diff
@@ -3946,13 +3946,11 @@ def reverse_sequence(input,
   >>> input = [[1, 2, 3, 4, 5, 0, 0, 0], [1, 2, 0, 0, 0, 0, 0, 0],
   ... [1, 2, 3, 4, 0, 0, 0, 0], [1, 2, 3, 4, 5, 6, 7, 8]]
   >>> output = reverse_sequence(input, seq_lens, seq_dim=1, batch_dim=0)
-  <tf.Tensor 'ReverseSequence_3:0' shape=(4, 8) dtype=int32>
-  >>> sess = tf.Session()
-  >>> print(sess.run(output))
-  [[0 0 5 4 3 2 1 0]
-  [2 1 0 0 0 0 0 0]
-  [3 2 1 4 0 0 0 0]
-  [5 4 3 2 1 6 7 8]]
+  <tf.Tensor: id=5, shape=(4, 8), dtype=int32, numpy=
```

Comment:
Can you please test this locally. I added the link to the guide in another comment before.

Reply:
Yes, I've tested it locally. 
> There is way to locally test the changes now: https://github.com/tensorflow/docs/blob/master/site/en/community/contribute/docs_ref.md#test-on-your-local-machine

But, when testing it on master (not my branch), it shows error message, `ValueError: line 5010 of the docstring for tensorflow.python.ops.array_ops has inconsistent leading whitespace: '  """'`; I am trying to fix, but can't make it yet. May I have your suggestions?

Reply:
I'll fix that issue, but this looks fine to me.